crates/node/src/evm.rs

```diff
@@ -11,7 +11,7 @@
 //! precompiles defined by [`odyssey_precompile`].
 
 use alloy_primitives::{Address, Bytes, TxKind, U256};
-use odyssey_precompile::secp256r1;
+use revm_precompile::secp256r1;
```

Comment:
secp is already available on optimism since fjord, which we build off of, so we don't need to manually add it

Reply:
it is secp256k1 that is available on the reth-optimism crate not secp256r1, not manually adding it would give build errors

Reply:
@onbjerg